src/pages/Account/Venue/VenueMapEdition/DraggableSubvenue.tsx

```diff
@@ -146,18 +145,22 @@ const styles: Record<string, CSSProperties> = {
     height: "100%",
     display: "flex",
     position: "relative",
+    border: "1px solid white",
```

Comment:
This `styles` syntax is legacy and should be refactored. Can you please add `@debt` to it.

Reply:
fixed in 3874ad43e2155b00a686bba8122dd8f5041a5fb3